File: wallet/src/main/java/com/worldventures/wallet/analytics/wizard/PinWasSetAction.java
package com.worldventures.wallet.analytics.wizard;

import com.worldventures.janet.analytics.AnalyticsEvent;
import com.worldventures.core.service.analytics.Attribute;
import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.wallet.analytics.WalletAnalyticsAction;

@AnalyticsEvent(action = "wallet:setup:Step 7:Your PIN is set",
                navigationState = true,
                trackers = AdobeTracker.TRACKER_KEY)
public class PinWasSetAction extends WalletAnalyticsAction {

   @Attribute("cardsetupstep7") final String cardsetupstep7 = "1";
}

File: wallet/src/main/java/com/worldventures/wallet/analytics/wizard/PinWasSetAction.kt
package com.worldventures.wallet.analytics.wizard

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.Attribute
import com.worldventures.core.service.analytics.AdobeTracker
import com.worldventures.wallet.analytics.WalletAnalyticsAction

@AnalyticsEvent(action = "wallet:setup:Step 7:Your PIN is set", navigationState = true, trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class PinWasSetAction : WalletAnalyticsAction() {

   @Attribute("cardsetupstep7") internal val cardsetupstep7 = "1"
}
